apps/rush-lib/src/logic/Git.ts

```diff
@@ -7,20 +7,33 @@ import * as child_process from 'child_process';
 import { PublishUtilities } from './PublishUtilities';
 
 export class Git {
```

Comment:
>Git [](start = 13, length = 3)

(possibly out scope) Suggest to either rename this to PublishingGit and start a new Git class that's actually useful.  Or alternatively remove the "targetBranch" from the constructor and make it a required parameter for each of the functions that needs it.